main.tf

```diff
@@ -1,6 +1,10 @@
 module "ecr_module"{
     source = "git@github.com:xerris/aws-modules.git//ecr"
-    ecr_name = "${var.env}-${var.ecr_name}"
+    ecr_name = var.ecr_name
```

Comment:
the env var is mandatory, you shouldn't remove it, we use it to identify the environment easier

Reply:
I will try to revert the env variable back on that line